src/index.html

```diff
@@ -1,14 +1,37 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div id="qa-block" class="first block">
+      <div class="second block">
+        <div class="third block">
+          In programming languages, a closure is a technique for implementing lexically scoped name binding in a language with first-class functions. Operationally, a closure is a record storing a function together with an environment.
+        </div>
```

Comment:
max 80 symbols for a line, it's should be okay to see the whole text without scrolling
```suggestion
        <div class="third block">
          In programming languages, a closure is a technique for implementing lexically scoped name binding in a
          language with first-class functions. Operationally, a closure is a record storing a function together with an
          environment.
        </div>
```